src/logger.py

```diff
@@ -0,0 +1,44 @@
+import logging
+from rich.logging import RichHandler
+
+FORMAT = "%(message)s"
+
+class Logger:
+    def __init__(self):
+        self.__FORMAT = "%(message)s"
+        logging.basicConfig(
+                level="NOTSET", format=FORMAT, datefmt="[%X]", handlers=[RichHandler(rich_tracebacks=True)]
+                )
+        self.__logger = logging.getLogger("rich")
```

Comment:
why use "rich" as logger name? It should point to where in application it originated. We can use "main" or "logger" (__name__)

Reply:
done. Btw should I add module level logging, or a single global logger is good enough?

Reply:
Yeah global logger is all fine. We just require ability to reduce log to INFO level if needed